beancount/parser/parser.py

```diff
@@ -258,10 +258,11 @@ def decorator(fun):
         filename = inspect.getfile(fun)
         lines, lineno = inspect.getsourcelines(fun)
 
-        # decorator line + function definition line (I realize this is largely
-        # imperfect, but it's only for reporting in our tests) - empty first line
-        # stripped away.
-        lineno += 1
+        # Skip over decorator invocation and function definition. This
+        # is imperfect as it assumes that each consumes exactly one
+        # line, but this is by far the most common case, and this is
+        # mainly used in tets, thus it is good enough.
```

Comment:
test, not tets

Reply:
You did not give me time to fix the typo :)